go.mod

```diff
@@ -95,6 +95,7 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 	github.com/Masterminds/goutils => github.com/Masterminds/goutils v1.1.1
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/mikefarah/yaml/v2 => gopkg.in/yaml.v2 v2.4.0 // Fix module path mismatch
```

Comment:
When the path mismatch happened? I'm wondering why the previous code without this can work..

Reply:
Did you try to `make build` in mac? I have this problem.